### DISCUSSION TITLE
Renault Zoe ZE40 Android Auto

### DISCUSSION THREAD

**Discussion by davidwkerr** [cedricp/ddt4all#1144] [General]
I've just successfully enabled AA on my Zoe using the ECU Tweaker version (on Pixel phone). Most people use the PC Windows version.

Worked quickly and brilliantly.

A big "Thank You" to Cedric and other developers.

Having AA will be great when 3G is turned off in Australia at the end of August.

Regds,
David

**Comment by morse90** [comment]
Is there any trick to it @davidwkerr ? Did you use a bluetooth or wirless adapter? I am just about to give it a go on our Zoe ZE40.

**Reply by AndyIreland** [reply]
you need to wire up a cable especially or make adjustments to the elm327 pins so you can read and write to the rlink1 through the OBD2 socket @morse90 

https://www.gps-rlink.com/r-link-1/activate-android-auto-waze-on-r-link-1/

**Reply by davidwkerr** [reply]
Yes, you will need the modified OBD cable. I used a BT dongle with that. Have a look around and you will find all the information you need. I have re-programmed Zoes with a Linux laptop and also Android phone.

**Reply by martinwetterstedt** [reply]
I'm struggeling with the DDT4ALL, do you think it is easier to use the ECU Tweaker? Are there any step-by-step instructions for that anywhere posted?

**Reply by AndyIreland** [reply]
yeah,have you made up the special lead to read the rlink through the OBD2 socket?

https://www.gps-rlink.com/r-link-1/activate-android-auto-waze-on-r-link-1/

**Reply by martinwetterstedt** [reply]
Yes!

**Comment by etvoorde** [comment]
Thanks to Cedric also from my side! ECU Tweaker also enabled Android Auto on my Renault Zoe!
